rllib/core/learner/tests/test_learner_group.py

```diff
@@ -128,7 +128,7 @@ def test_learner_group_local(self):
 
     def test_update_multigpu(self):
         fws = ["torch", "tf2"]
-        scaling_modes = REMOTE_SCALING_CONFIGS.keys()
+        scaling_modes = ["multi-gpu-ddp", "remote-gpu"]
```

Comment:
@avnishn Can you add explanations on why these changes are made? `REMOTE_SCALING_CONFIGS.keys()` --> `["multi-gpu-ddp", "remote-gpu"]`?

Reply:
We can reduce the number of tests that we need to run while getting full test coverage by doing this.